risc0/zkp/src/prove/soundness.rs

```diff
@@ -0,0 +1,292 @@
+use crate::{hal::Hal, taps::TapSet, FRI_FOLD, FRI_MIN_DEGREE, INV_RATE};
+use risc0_core::field::{baby_bear, ExtElem};
+
+/// Johnson's bound
+const M: f32 = 16.0;
+
+/// Rate
+const RHO: f32 = 1.0 / crate::INV_RATE as f32;
+
+/// η in Conjecture 8.4 [BCIKS21]
```

Comment:
Seems not immediately obvious how to track down a reference for BCIKS21. 

Maybe makes sense to use an eprint link instead? or i guess in this case mb [this link? ](https://dl.acm.org/doi/pdf/10.1145/3614423)